packages/jest-utils/tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
```

Comment:
Can we just have everything in the tsconfig.eslint.json instead of extending it from another file?

Reply:
this is how its done in other packages